index.js

```diff
@@ -89,4 +97,25 @@ module.exports = function (processors, options) {
   }
 
   return stream
+})
+
+
+function withConfigLoader(cb) {
+  return function (plugins, options) {
+    if (typeof plugins === 'undefined') {
+      return cb(postcssLoadConfig)
```

Comment:
I haven't found anything about `ext` and `dir` in the [readme](https://github.com/michael-ciniawsky/postcss-load-config). They can be calculated from `from` and `to` that are there. https://github.com/postcss/gulp-postcss/pull/108/files#diff-168726dbe96b3ce427e7fedce31bb0bcR36